tests/device_tests/test_msg_cardano_get_address.py

```diff
@@ -26,24 +26,43 @@
 @pytest.mark.cardano
 @pytest.mark.skip_t1  # T1 support is not planned
 @pytest.mark.parametrize(
-    "path,expected_address",
+    "path,protocol_magic,expected_address",
     [
+        # data from https://iancoleman.io/bip39/
```

Comment:
afait data isn't from there

Reply:
Honestly, I don't know. The comment was at the start of the test before, so I moved it up since it definitely isn't true for all the test cases now. Should I remove it since we aren't sure?

Reply:
Remove it, won't be true for Shelley anyway.